src/Modules/OrchardCore.Commerce/Controllers/PaymentController.cs

```diff
@@ -207,9 +207,10 @@ public async Task<IActionResult> SuccessPost(string orderId)
 
         await _contentManager.UpdateAsync(order);
 
-        if (_workflowManager != null)
+        if (_workflowManagers != null)
```

Comment:
If no `IWorkflowManager` services are registered `IEnumerable<IWorkflowManager>` will never be `null`, it will be empty. You can either do `if (_workflowManagers.FirstOrDefault() is { } workflowManager)` here, or `_workflowManager = workflowManagers.FirstOrDefault();` in the constructor.